### PR TITLE
Add new story

### PR DESCRIPTION
**Pull request by karestrepo**

### CODE REVIEW

**Review by Mose-Skolnick — DISMISSED** [review]

File: content/topics/sdg2-zero-hunger/stories/Katerine-Restrepo-1/index.md [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+## **Varibles Used :**
+| Variable Name | Description |
+| :-- | :-: |
+|<b>sdg2_2_cmc</b>|Chronic malnutrition in children (< 5 years), 2016 (%)|
+|<b>sdg2_2_oww</b>|Obesity in women (15-49 years), 2016 (%)|
+|<b>sdg3_2_mrc</b>|Children mortality rate in (< 5 years), 2016 (per. 1,000 live births)|
+
+
+
+{{< paragraph class="large" >}}
```
Comment (GeoffreyXue) [inline content/topics/sdg2-zero-hunger/stories/Katerine-Restrepo-1/index.md:19]:
```suggestion
```

File: content/topics/sdg2-zero-hunger/stories/Katerine-Restrepo-1/index.md [hunk 2]
```diff
@@ -0,0 +1,39 @@
+---
+title: Does the infant mortality rate in Bolivia exhibit correlations with nutritional challenges? Could there be evidence of a Simpson's Paradox?
+author: Katerine Restrepo
+date: 2023-10-04
+data: "/data/GeoDS4Bolivia.geojson"
+deepnote: "https://deepnote.com/workspace/nagoya-university-102f-072bbb53-844b-47b1-aa0d-c8654cf05167/project/Nutritional-Inequality-b5441f3b-b9ff-452a-a4cf-3a2ce6025e8d/notebook/Infant%20Mortality%20and%20Nutrition%20in%20Bolivia-84beb61d25a44593b6f174613c80f259"
```
Comment (GeoffreyXue) [inline content/topics/sdg2-zero-hunger/stories/Katerine-Restrepo-1/index.md:6]:
```suggestion
links:
    deepnote: "https://deepnote.com/workspace/nagoya-university-102f-072bbb53-844b-47b1-aa0d- c8654cf05167/project/Nutritional-Inequality-b5441f3b-b9ff-452a-a4cf-3a2ce6025e8d/notebook/Infant%20Mortality%20and%20Nutrition%20in%20Bolivia-84beb61d25a44593b6f174613c80f259"
```

**Review by GeoffreyXue — CHANGES_REQUESTED** [review]
Looks good! Just a couple of minor cleanup changes

File: content/topics/sdg2-zero-hunger/stories/Katerine-Restrepo-1/index.md (hunk 2 above)
Comment (GeoffreyXue) [reply content/topics/sdg2-zero-hunger/stories/Katerine-Restrepo-1/index.md:6]: The motivation for changing this from just the deepnote to be prefixed with links is due to #96, where the `links` front matter was introduced, ideally replacing the `deepnote` link that is currently available. More explanation can be found on the PR above.

**Comment by GeoffreyXue** [comment]
One final change @karestrepo, could you please rename the file to `story.md`? It was a more recent change to differentiate stories for now, since Hugo treats files named `index.md` in a particular way, which is not really the intended case for stories.

More on this: https://gohugo.io/content-management/page-bundles/

File: content/topics/sdg2-zero-hunger/stories/Katerine-Restrepo-1/story.md [hunk 3]
```diff
@@ -0,0 +1,39 @@
+---
+title: Does the infant mortality rate in Bolivia exhibit correlations with nutritional challenges? Could there be evidence of a Simpson's Paradox?
+author: Katerine Restrepo
+date: 2023-10-04
+data: "/data/GeoDS4Bolivia.geojson"
+links:
+    deepnote: "https://deepnote.com/workspace/nagoya-university-102f-072bbb53-844b-47b1-aa0d- c8654cf05167/project/Nutritional-Inequality-b5441f3b-b9ff-452a-a4cf-3a2ce6025e8d/notebook/Infant%20Mortality%20and%20Nutrition%20in%20Bolivia-84beb61d25a44593b6f174613c80f259"
```
Comment (GeoffreyXue) [inline content/topics/sdg2-zero-hunger/stories/Katerine-Restrepo-1/story.md:7]:
```suggestion
  deepnote: "https://deepnote.com/workspace/nagoya-university-102f-072bbb53-844b-47b1-aa0d- c8654cf05167/project/Nutritional-Inequality-b5441f3b-b9ff-452a-a4cf-3a2ce6025e8d/notebook/Infant%20Mortality%20and%20Nutrition%20in%20Bolivia-84beb61d25a44593b6f174613c80f259"
```
Sorry, added two extra spaces here

**Review by GeoffreyXue — CHANGES_REQUESTED** [review]
One that thing that was on me

File: content/topics/sdg2-zero-hunger/stories/Katerine-Restrepo-1/story.md [hunk 4]
```diff
@@ -0,0 +1,39 @@
+---
+title: Does the infant mortality rate in Bolivia exhibit correlations with nutritional challenges? Could there be evidence of a Simpson's Paradox?
+author: Katerine Restrepo
+date: 2023-10-04
+data: "/data/GeoDS4Bolivia.geojson"
+links:
+  deepnote: "https://deepnote.com/workspace/nagoya-university-102f-072bbb53-844b-47b1-aa0d- c8654cf05167/project/Nutritional-Inequality-b5441f3b-b9ff-452a-a4cf-3a2ce6025e8d/notebook/Infant%20Mortality%20and%20Nutrition%20in%20Bolivia-84beb61d25a44593b6f174613c80f259"
```
Comment (GeoffreyXue) [inline content/topics/sdg2-zero-hunger/stories/Katerine-Restrepo-1/story.md:7]:
```suggestion
  deepnote: https://deepnote.com/workspace/nagoya-university-102f-072bbb53-844b-47b1-aa0d- c8654cf05167/project/Nutritional-Inequality-b5441f3b-b9ff-452a-a4cf-3a2ce6025e8d/notebook/Infant%20Mortality%20and%20Nutrition%20in%20Bolivia-84beb61d25a44593b6f174613c80f259
```

Hmm, still not rendering. One more try?

**Comment by GeoffreyXue** [comment]
Ah I see, it seems like the branch is just out of date with master. I'll update the PR really quick (just doing a `git merge origin/main`

**Comment by GeoffreyXue** [comment]
> Ah I see, it seems like the branch is just out of date with master. I'll update the PR really quick (just doing a `git merge origin/main`

For more clarification on this - this branch wasn't up to date with the changes I made in previous commits, so the links weren't rendering. I've now pulled in the changes since this branch was created, and hopefully it will now correctly render the links.

**Review by GeoffreyXue — APPROVED** [review]
Looks like it's working now!

**Comment by karestrepo** [comment]
the  deepnote link is not working

**Pull request merged**